packages/core/src/event-listeners/grant.ts

```diff
@@ -69,6 +69,7 @@ const grantTypeToExchangeByType: Record<GrantType, token.ExchangeByType> = {
   [GrantType.AuthorizationCode]: token.ExchangeByType.AuthorizationCode,
   [GrantType.RefreshToken]: token.ExchangeByType.RefreshToken,
   [GrantType.ClientCredentials]: token.ExchangeByType.ClientCredentials,
+  [GrantType.OrganizationToken]: token.ExchangeByType.RefreshToken,
```

Comment:
Could you add a comment here? It is confusing that "OrganizationToken" key with value "RefreshToken"